### PR TITLE
Axis reader tests

### PR DESCRIPTION
**Pull request by singhraman4282**
**What's this do?**
Adds tests for ZlibDecompressor and NYPLRSACypher

**Why are we doing this? (w/ JIRA link if applicable)**
N/A

**How should this be tested? / Do these changes have associated tests?**
Cannot be QA tested

**Dependencies for merging? Releasing to production?**
N/A

**Does this include changes that require a new SimplyE build for QA?**
N/A

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
Raman Singh verified it

### CODE REVIEW

**Review by ErnestFan — APPROVED** [review]

File: SimplifiedTests/NYPLAxisTests/NYPLRSACypherTests/NYPLRSACypherTests.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//  Created by Raman Singh on 2021-05-28.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import XCTest
+@testable import SimplyE
+
+class NYPLRSACypherTests: XCTestCase {
+  
+ let cypher = NYPLRSACypher()!
```
[the remark this answers is not in the document]
Comment (singhraman4282) [reply SimplifiedTests/NYPLAxisTests/NYPLRSACypherTests/NYPLRSACypherTests.swift]: Done

**Comment by singhraman4282** [comment]
> tests look fine, but I'm wondering if we got clearance to use and check in those book chapter samples?

@leonardr can we get your opinion on this? As of now I'm using a real chapter from a real book with a real AES key

**Comment by leonardr** [comment]
Thanks for looping me in. You're right, we can't use actual book contents as test data (metadata is fine). Having encrypted versions of that data is also bad, given that running the unit tests presumably decrypts the data.

Also, I don't know what's in SimplifiedTests/NYPLAxisTests/NYPLRSACypherTests/AESKeyData.txt but the filename makes me a little concerned.

General rules:

* No real book text in tests unless the text is very short (a paragraph) or public domain. And if you just need a paragraph, you can get a public domain paragraph.
* No real artifacts of a secret encryption process in a public repository. (Private repo is fine.)

I think this is what I'd like to see here:

* Compress some fake data or public domain content and use that to test the decompression algorithm. You wrote the code and the tests using real data, so we can swap out fake data and be confident that the algorithm still works.

* Generate an encryption key (rather than taking a real one, even a temporary one, from Axis) and encrypt the fake content. Then use that key to decrypt the fake content in the test, and make it clear in comments that this is not a real key or real content.

Let me know if you have trouble coming up with the compressed and encrypted test data and encryption key; I may be able to make some time to generate it for you.

**Comment by singhraman4282** [comment]
@leonardr @ettore thank you for your feedback. The encrypted, decrypted, userkey, compressed, and decompressed files have been replaced with dummy data created locally and do not contain any information that one would obtain from a real book. The branch has also been rebased.

**Pull request merged**